lib/connectable.rb

```diff
@@ -0,0 +1,9 @@
+module Connectable
+
+  def conn(url, api_info)
+    Faraday.new url do |conn|
+      conn.params[api_info[:name]] = api_info[:key]
+    end
+  end
```

Comment:
`api_info[:name]` the name expected for the api key
`api_info[:key]` the actual api key